--- a/drivers/pinctrl/mediatek/Makefile	2024-09-11 11:06:42.089258000 +0200
+++ b/drivers/pinctrl/mediatek/Makefile	2024-09-11 11:07:20.390028384 +0200
@@ -17,7 +17,7 @@
 obj-$(CONFIG_PINCTRL_MT7622)	+= pinctrl-mt7622.o
 obj-$(CONFIG_PINCTRL_MT7623)	+= pinctrl-mt7623.o
 obj-$(CONFIG_PINCTRL_MT7629)	+= pinctrl-mt7629.o
-obj-$(CONFIG_PINCTRL_MT7986)	+= pinctrl-mt7981.o
+obj-$(CONFIG_PINCTRL_MT7981)	+= pinctrl-mt7981.o
 obj-$(CONFIG_PINCTRL_MT7986)	+= pinctrl-mt7986.o
 obj-$(CONFIG_PINCTRL_MT7988)	+= pinctrl-mt7988.o
 obj-$(CONFIG_PINCTRL_MT8167)	+= pinctrl-mt8167.o
